### ISSUE TITLE
@glimmer/component not included in dependencies

### ISSUE THREAD

**Issue by gzurbach** [shipshapecode/ember-flatpickr#615]
Hi there,

I'm in the process of updating `ember-flatpick` from `2.16.2` to `3.0.1`. 

I am getting the following runtime error:

> Error: Could not find module `@glimmer/component` imported from `ember-flatpickr/components/ember-flatpickr`.

I am currently running Ember 3.12. Is there a minimum version of Ember required to run `3.0.1`?

Thanks a lot for all the good work! 👍

**Comment by rwwagner90** [comment]
@gzurbach I think the main issue here is we should add `@glimmer/component` to dependencies instead of devDependencies. I'm unsure what versions it works with, but I would guess Ember 3.x would yield the best results. Are there issues holding you back from updating to Ember 3.x?

**Comment by gzurbach** [comment]
@rwwagner90 I'm actually using Ember 3.12 and not 2.12, my bad. I edited my comment above. I _think_ `@glimmer/component` was brought in Ember 3.16, which is where I should be heading?

**Comment by rwwagner90** [comment]
@gzurbach You can add `@glimmer/component` to your dependencies and it should work fine until we make the change here.

**Comment by gzurbach** [comment]
@rwwagner90 Thanks for getting back to me. FYI I ended up updating the app to Ember 3.16 and it does come with `@glimmer/component`.

**Comment by rwwagner90** [comment]
@gzurbach yeah, it was added to the default blueprints at one point, but someone could easily remove it from their dependencies if they didn't want it, and then this addon would be broken, so we do still need to include it in depdendencies.

**Comment by LuisAverhoff** [comment]
@gzurbach Ember 3.13 was the first version to introduce glimmer components.

@rwwagner90 Since this project introduced glimmer components, it is probably best to update the ember-try and change this part of the file.

```
 {
        name: 'ember-lts-3.16',
        npm: {
          devDependencies: {
            'ember-source': '~3.16.0'
          }
        }
      },
```

to

```
 {
        name: 'ember-3.13',
        npm: {
          devDependencies: {
            'ember-source': '~3.13.0'
          }
        }
      },
```

since that is when glimmer components were first introduced. Likewise, the readme in this project should specify the minimum requirements needed to use this library i.e at least ember 3.13.

**Comment by rwwagner90** [comment]
@LuisAverhoff we are just following the recommended minimum testing versions from Ember try, so we don't get stale. I don't think we want to add tests for 3.13.

**Comment by LuisAverhoff** [comment]
@rwwagner90 Alright I'll remove all the tests for 3.13 but I propose we still leave in the `readme` that the minimum is `3.13` since anything below that will not work because glimmer components were not included in the default blueprint until `3.13`.

**Comment by rwwagner90** [comment]
@LuisAverhoff we had `3.16` before as that's the lowest version we're actively testing. I think we should keep that.
